src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,473 @@
+>
+  Control plane components on [OpenShift](http://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) 4.x use endpoint URLs that require SSL and service account based authentication. Therefore, [the default](/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#discover-nodes-components) endpoint URLs can not be used.
+
+  To configure control plane monitoring on OpenShift, uncomment the following environment variables in the [customized manifest](/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure#customized-manifest). URL values are pre-configured to the default base URLs for the control plane monitoring metrics endpoints in [OpenShift](http://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) 4.x.
+
+  ```
+  - name: "SCHEDULER_ENDPOINT_URL"
+    value: "https://localhost:10259
+  - name: "ETCD_ENDPOINT_URL"
+    value: "https://localhost:9979"
+  - name: "CONTROLLER_MANAGER_ENDPOINT_URL"
+    value: "https://localhost:10257"
+  - name: "API_SERVER_ENDPOINT_URL"
+    value: "https://localhost:6443"
+  ```
+
+  <Callout variant="important">
+    Even though the custom `ETCD_ENDPOINT_URL` is defined, ETCD requires HTTPS and mTLS authentication to be configured. For more on configuring mTLS for ETCD in OpenShift, see [Set up mTLS for ETCD in OpenShift](#mtls-how-to-openshift).
+  </Callout>
+
+  <Callout variant="important">
+    When installing through Helm `openshift`, specify the config to automatically include these endpoints. Setting `openshift.enabled=true` and `openshift.version="4.x"` will include the secure endpoints and enable the `/var/run/crio.sock` runtime.
+  </Callout>
+</Collapser>
+</CollapserGroup>
+
+### Set up mTLS for ETCD in OpenShift [#mtls-how-to-openshift]
+
+Follow these instructions to set up mutual TLS authentication for ETCD in [OpenShift](http://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) 4.x:
+
+1. Export the ETCD client certificates from the cluster to an opaque secret. In a default managed OpenShift cluster, the secret is named `kube-etcd-client-certs` and it is stored in the `openshift-monitoring` namespace.
+
+   ```shell
+   kubectl get secret/kube-etcd-client-certs -n openshift-monitoring -o yaml > etcd-secret.yaml
+   ```
+
+2. Open the secret file and change the keys:
+   * Rename the **certificate authority** to `cacert`.
+   * Rename the **client certificate** to `cert`.
+   * Rename the **client key** to `key`.
+
+3. Optional: change the secret name and namespace to something meaningful.
+
+4. Remove these unnecessary keys in the metadata section:
+   * `creationTimestamp`
+   * `resourceVersion`
+   * `selfLink`
+   * `uid`
+
+5. Install the manifest with its new name and namespace:
+
+   ```shell
+   kubectl apply -n newrelic -f etcd-secret.yaml
+   ```
+
+6. Configure the integration to use the newly created secret as described in [the mtls section](#mtls).
```

Comment:
Could we just point the integration to this already existing secret, and get rid of this whole section?

Might be related to https://github.com/newrelic/nri-kubernetes/issues/342